src/presentation/controllers/action-controller.ts

```diff
@@ -25,10 +25,10 @@ export class ActionController extends BaseController {
   }
 
   @PutRequest('/:actionId/rundowns/:rundownId')
-  public async executeAction(reg: Request, res: Response): Promise<void> {
+  public async executeAction(req: Request, res: Response): Promise<void> {
```

Comment:
A general thing for all of us. It would be nice to have controller parameters be `request` and `response`.

Reply:
Will make a small PR changing this on all our controllers, after this is merged.